pkg/proxy/iptables/proxier.go

```diff
@@ -725,42 +764,33 @@ func (proxier *Proxier) syncProxyRules() {
 			args = append(args,
 				"-m", protocol, "-p", protocol,
 				"-j", "DNAT", "--to-destination", endpoints[i])
-			writeLine(rulesLines, args...)
+			writeLine(natRules, args...)
 		}
 	}
 
 	// Delete chains no longer in use.
-	for chain := range existingChains {
```

Comment:
@thockin I did this originally because I wasn't sure which chain we'd be putting the REJECT rules onto, i'll revert it.